### PR TITLE
Hosting/ANCM doc update

### PR DESCRIPTION
**Pull request by GuardRex**
Fixes https://github.com/aspnet/Docs/issues/1965

### CODE REVIEW

**Comment by shirhatti** [comment]
Besides that one comment. Looks good to me

File: aspnet/aspnetcore/hosting/aspnet-core-module.md [hunk 1]
```diff
@@ -5,37 +5,36 @@ uid: hosting/aspnet-core-module
...
-| startupTimeLimit        |<p>Optional integer attribute.</p><p>Duration in seconds for which the the handler will wait for the executable or script to start a process listening on the port. If this time limit is exceeded, the handler will kill the process and attempt to launch it again **startupRetryCount** times.</p><p>The default valueis 120.</p>|
-| shutdownTimeLimit       |<p>Optional integer attribute.</p><p>Duration in seconds for which the the handler will wait for the executable or script to gracefully shutdown when the *app_offline.htm* file is detected.</p><p>The default valueis 10.</p>|
-| rapidFailsPerMinute     |<p>Optional integer attribute.</p><p>Specifies the number of times the process specified in **processPath** is allowed to crash per minute. If this limit is exceeded, the handler will stop launching the process for the remainder of the minute.</p><p>The default valueis 10.</p>|
+| arguments               |<p>Optional string attribute.</p><p>Arguments to the executable or script specified in **processPath**.</p><p>The default value is an empty string.</p>|
```
Comment (shirhatti) [inline aspnet/aspnetcore/hosting/aspnet-core-module.md]: 👍

Comment (GuardRex) [inline aspnet/aspnetcore/hosting/aspnet-core-module.md]: `processPath` has a similar statement ...

> Path to the executable or script that will launch a process listening for HTTP requests. ...

Change that one, too?

> Path to the executable that will launch a process listening for HTTP requests. ...

Comment (GuardRex) [inline aspnet/aspnetcore/hosting/aspnet-core-module.md]: I saw the word "script" in a few spots; so in the latest commit, I just pulled that word everywhere for consistency.

**Comment by GuardRex** [comment]
@shirhatti The latest commit gets the `recycleOnFileChange` language in place. Unless there's anything else, I'm good with it for now and will review the whole doc again at 1.1 time.

**Pull request merged**